content/md/first-steps/manage-families.md

```diff
@@ -268,6 +278,18 @@ To view the family versions:
 Deleting a family does not delete the attributes it includes, or the products associated with it. The attributes are always available and the products keep these very same attributes if they have values: they will become “optional” and the products with no family will have no longer completeness. The values are though maintained: there is no loss of product information.
 :::
 
+::: warning
+A family with family variants cannot be deleted. You need to first [delete its family variants](/articles/manage-your-families.html#delete-a-family-variant-since-22).
```

Comment:
AFAIK, we should say "you first need to"

Reply:
ah yes :)